shared/core/src/wasmJsMain/kotlin/com/shub39/grit/core/settings/presentation/ui/section/LookAndFeelPage.wasmJs.kt

```
@@ -0,0 +1,163 @@
+import grit.shared.core.generated.resources.check_circle
+import grit.shared.core.generated.resources.palette
+import grit.shared.core.generated.resources.palette_style
+import org.jetbrains.compose.resources.stringResource
+import org.jetbrains.compose.resources.vectorResource
+
+@Composable
+actual fun MaterialYouToggle(
+    isUserSubscribed: Boolean,
+    isMaterialYou: Boolean,
+    onClick: (Boolean) -> Unit,
+) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)